docs/reference-guides/theme-json/theme-json-reference-v1.md

```diff
@@ -0,0 +1,142 @@
+# Version 1 Reference
+
+Theme.json version 2 has been released, see the [theme.json migration guide](/docs/reference-guides/theme-json/theme-json-migrations.md#migrating-from-v1-to-v2) for updating to the latest version.
+
+<!-- START TOKEN Autogenerated - DO NOT EDIT -->
+## Settings
+
+
+### border
+
+Settings related to borders.
+
+| Property  | Type   | Default | Props  |
+| ---       | ---    | ---    |---   |
+| customColor | boolean | false |  |
```

Comment:
As per the `ALLOWED_SETTINGS` in the `5.8` [branch](https://github.com/WordPress/wordpress-develop/blob/5.8/src/wp-includes/class-wp-theme-json.php#L181), we only supported `customRadius` in WordPress 5.8.